portfolio/src/main/webapp/script.js

```diff
@@ -171,9 +171,14 @@ function createCommentElement(comment) {
   const timestampString = new Date(comment.timestamp).toString();
   timeElement.innerText = timestampString;
 
+  const scoreElement = document.createElement('div');
+  scoreElement.innerText = 'Sentiment score (Scale from -1 to 1): ' +
+      comment.sentimentScore.toString();
```

Comment:
you should use toFixed instead to restrict the number of digits after decimal (https://www.w3schools.com/JSREF/jsref_tofixed.asp)